### DISCUSSION TITLE
[Unstable] Gravity ammo requires 2 hands

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14389] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/a977c93e-0598-4f9b-82e4-f92b494187a4)
![image](https://github.com/user-attachments/assets/a56e4995-4b11-4bbd-a1ba-965956fa26af)

These should be held in 1 hand for consistency with all other ammunition types.

### Reproduction steps

1. Get gravity ammunition 
2. Gravity ammunition very heavy, need two hands to hold it

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/bd5bc2363781cce0a06c62169e21cfc29b61f339 (commit in our private repo), fix coming up in the next unstable build.
